.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Tests
+
+on: 
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs: 
+  unit-test:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with: 
+          node-version: "14"
+      - name: Setup
```

Comment:
Any reason why we can't just do `npm install` from the root? and not install the individual dependencies?